finetune/base.py

```diff
@@ -265,6 +267,11 @@ def get_estimator(self, force_build_lm=False):
             train_distribute=distribute_strategy,
             keep_checkpoint_max=1
         )
+        return conf, distribute_strategy, config
+
+
+    def get_estimator(self, force_build_lm=False):
+        conf, distribute_strategy, config = self._get_estimator_config()
```

Comment:
Code style nitpicking but in general I think it's bad practice to have methods that return tuples of results in no particular order.  My preference is to return dictionaries w/ named keys (something like "config_proto", "run_config", and "distribution_strategy" in this case).